src/datoso_plugin_internetarchive/fetch.py

```diff
@@ -5,6 +5,7 @@
 from concurrent.futures import ThreadPoolExecutor
 import zipfile
 from pathlib import Path
+from urllib.parse import urljoin
```

Comment:
```suggestion
from datoso.helpers import RequestUtils, downloader
```
and remove `from datoso.helpers import downloader`